cmd/istioctl/main.go

```diff
@@ -565,6 +565,7 @@ func readInputs() ([]model.Config, []crd.IstioKind, error) {
 }
 
 // Print a simple list of names
+// nolint: errcheck, gas
```

Comment:
we probably shouldn't do that in non test files

Reply:
megacheck replaces several older linters. we lose performance I think (may time out).